### ISSUE TITLE
Balancer Basics

### ISSUE THREAD

**Issue by hwhelchel** [TeleportXYZ/TRIP-Guides#3]
### What's missing from the TRIP Guides

We need a simple overview of the basics of guiding others to use TRIP.

### Expected content

Topics guides might be interested in:

- How sending invitations works
- How accepting invitations works
- Effective ways to recruit riders
- Effective ways to recruit drivers
- The importance of recruiting people with large audiences (who can recruit lots of people in your chain)

### Actual content

Right now this is a stub page.

**Comment by hwhelchel** [comment]
@matt-brigante in terms of what I wrote above in the issue, don't look at that as specific instructions on what you should add here. Rather, I just wanted to model the format of an example issue for you to give you a framework to use when creating your own issues here.

As a first step, make a comment here on this issue as if you're creating a new issue and propose your changes. Then anyone on the team interested can join the discussion, get aligned, and then you can make your change request in GitBook.

Thanks!

**Comment by hwhelchel** [comment]
@tlow22 I think the work you're doing here https://app.gitbook.com/o/0WSd8UiSeH2xhfJrSbUr/s/NwtZr1A5O8eLe1w4Pmm4/~/changes/105/explorer/signup-progress should live on the balancer basics page for now.

The Explorer Basics page should describe the current state of the TRIP Explorer.

The Balancer Basics page should describe the current state of balancer mechanics and unlocking cities.

**Comment by hwhelchel** [comment]
@tlow22 maybe we have two sections on the Balancer Basics page:

- Atomic Network Estimates
- Atomic Network Progress

Or maybe we can just get rid of the Atomic Network Estimates section entirely since it's embedded in the progress table.

**Comment by tlow22** [comment]
I'll keep it in there for now, until progress table is on parity in terms the cities listed. Working towards that, but yes, once that happens, we should deprecate the Atomic Network Estimates?

**Comment by hwhelchel** [comment]
@tlow22 what's stopping the progress table from having parity with the cities listed? Is it just a bit of manual data entry or something else? If it's just a bit of manual data entry, let's just knock it out and get rid of the old page as part of shipping this.

**Comment by r-woosley** [comment]
@tlow22 I have a couple of question regarding the chart. 

1. Is this data live? If not, when should we expect updates? Maybe an updated timestamp.
2. Are these numbers implying the whole city as the service area or a predefined area yall have selected?

**Comment by tlow22** [comment]
> @tlow22 I have a couple of question regarding the chart.
> 
> 1. Is this data live? If not, when should we expect updates? Maybe an updated timestamp.
> 2. Are these numbers implying the whole city as the service area or a predefined area yall have selected?

1. Currently, it is scheduled in 1 hour updates. Yep, a timestamp to come!
2. These are pre-defined based on some polygons that OpenStreetMaps provides since we use them in our simulation for the estimates. But we're not really a monolithic operator with solidified geofences, so these definitions are not absolute. 

Next step is to fuse some of these "sub-cities"  into the larger encompassing metro. At one point, I was deciding which Austin suburb to fuse into Austin but realized that there's some subjectivity that the community can definitely help drive here. @hwhelchel this also relates to your question!

**Comment by tlow22** [comment]
@hwhelchel both the riders and drivers table has now been expanded. On the rider table, we now have riders in more areas than we have for goal estimates. 

@r-woosley timestamp has been added

**Comment by r-woosley** [comment]
@tlow22 Does the new table show each city that riders have signed up in or does it have to meet a certain population? Just curious because the Explorer shows several signups in suburbs of Austin, Tx, (Ex. Cedar Park, Kyle, Pflugerville) but not seeing those  cities listed.

**Comment by tlow22** [comment]
@r-woosley there is a population filter of 100,000 when counting the signups to a city. So in this case, the suburbs you mentioned are not included and get grouped under Austin. 

Do you think/prefer that these suburbs have their own counts?

**Comment by r-woosley** [comment]
@tlow22 I was curious how suburbs would be unlocked as the system becomes decentralized and cities automatically unlock once their atomic numbers are hit. 

If suburbs are tied into the the metro they surround what would be the methodology for them to eventually go live as they wont be in the coverage area when their respective metro launches? Round Rock is listed, but is also a suburb of Austin. Will it need to hit its unlock numbers to go live or will it unlock when the coverage area in Austin expands? 

Just want to make sure I'm clear on how unlocking will work in metros/suburbs and what suburbs tie to what metros.

**Comment by tlow22** [comment]
@r-woosley ah so suburbs and the metro they tie to are not gospel yet, and super open to feedback from community on modifying it locally for their region. For 1st iteration, we had to generate over a wide range of cities to be inclusive. 

There is some variability here because Operators eventually determine what coverage area they want, and starting coverage could be much smaller than existing competitors. 

Seems to me maybe more info needs to be offered as well when suburbs are grouped, in the form of "What counts towards your metro" on the table. 

What do you think?

**Comment by hwhelchel** [comment]
@tlow22

> both the riders and drivers table has now been expanded.

So does your new dashboard have the same number of cities as in the Atomic Network Estimates page?

If so, let's get rid of that page and just have one page for this data. Let's also move this into the Balancer Basics page instead of it being a separate page. Once this is done, can you close this issue here in GitHub as completed? Thank you!
